test/errors-spec.js

```diff
@@ -0,0 +1,339 @@
+/* global beforeEach: false */
+/* global jasmine: false */
+/* global expect: false */
+/* global it: false */
+/* global describe: false */
+
+var parseTree = require("../src/parseTree");
+var Settings = require("../src/Settings");
+
+var defaultSettings = new Settings({});
+
+beforeEach(function() {
+    jasmine.addMatchers({
+        toParseWithError: function(util, customEqualityTesters) {
```

Comment:
I think I would call this `toFailWithError`, since the other tests use a `toParse` matcher that checks that the parse succeeded.